agent/discovery/collector/client.go

```diff
@@ -19,6 +19,7 @@ import (
 
 type Client interface {
 	Publish(discoveryType string, payload interface{}) error
+	Heartbeat() error
```

Comment:
Just for reference, since I think it is relates. https://github.com/trento-project/trento/pull/441#discussion_r747884377 

As we already said, not necessary to act now